content/posts/2023-12-04-matura-bel-top-schools.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Малки общини постигат високи резултати на матурите
+date: 2023-12-04 00:00:00
+description: Анализ на представянето на матурите по БЕЛ в малките общини в България за 2023 г.      
+featured_image: /assets/img/pages/color-graph.png
+author: Данни за добро, Образование без раници
```

Comment:
* the key is `authors` , not `author`
this should be an array of keys from [people.json](https://github.com/data-for-good-bg/data-for-good-bg.github.io/blob/blog-post-bel-2023/data/people.json) 
Either just add yourself `isabella-taskova` or add an enrtry for dani and add both of you

Reply:
- people.json updated to add Teodora Atanasova and Daniel Cholakov, authors of the article
- Authors updated in markdown